SimPEG/electromagnetics/time_domain/simulation_1d_stitched.py

```diff
@@ -0,0 +1,208 @@
+import numpy as np
+from scipy import sparse as sp
+from ... import utils
+from ..base_1d_stitched import BaseStitchedEM1DSimulation
+from .simulation_1d import Simulation1DLayered
+from .survey import Survey
+from ... import maps
+from multiprocessing import Pool
+
+
+def run_simulation_time_domain(args):
```

Comment:
Why not expose the args? We could probably use a dataclass at some point to reduce the complexity.

Reply:
no particular reasons. my dirty implementation for multiprocessing.